Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -390,6 +390,18 @@ local function hsi_range_string(dev0, arg_hundreds_metric, arg_tens_metric, arg_
 	end
 end
 
+local function getCountermesureCounterString(arg_value)
```

Comment:
I think you have a typo in the method name here (**mesure** instead of **measure**)